cookbook/web_services/rest_api.rst

```diff
@@ -0,0 +1,29 @@
+.. index::
+    single: Web Services; REST API
+
+How to Create a REST Web Service in a Symfony2 Controller
+=========================================================
+
+A REST controller must be able to list, retrieve, create and update a resource.
```

Comment:
why a rest controller must be able?this is confusing, it's up to the controller / router what kind of verbs are supportet.